boards/nordic/nrf9280pdk/nrf9280pdk_nrf9280-memory_map.dtsi

```diff
@@ -0,0 +1,271 @@
+				reg = <0x0 DT_SIZE_K(2)>;
+			};
+
+			cpuapp_cpusec_ipc_shm: memory@800 {
+				reg = <0x800 DT_SIZE_K(2)>;
+			};
+
+			cpuapp_data: memory@1000 {
+				reg = <0x1000 DT_SIZE_K(512)>;
+			};
+		};
+
+		cpuapp_cpurad_ram0x_region: memory@2f0cf000 {
+			compatible = "nordic,owned-memory";
+			reg = <0x2f0cf000 DT_SIZE_K(4)>;
+			status = "disabled";
+			perm-read;
+			perm-write;
+			#address-cells = <1>;
+			#size-cells = <1>;
+			ranges = <0x0 0x2f0cf000 0x1000>;
+
+			cpuapp_cpurad_ipc_shm: memory@0 {
+				reg = <0x0 DT_SIZE_K(2)>;
+			};
+
+			cpurad_cpuapp_ipc_shm: memory@800 {
+				reg = <0x800 DT_SIZE_K(2)>;
+			};
+		};
+
+		/* TODO:
+		 * refactor after https://github.com/zephyrproject-rtos/zephyr/pull/72273
+		 * to use the `access` binding to specify shared memory ownership.
+		 */
+		cpuapp_cpucell_ram0x_region: memory@2f0d0000 {
+			reg = <0x2f0d0000 DT_SIZE_K(36)>;
+			status = "disabled";
+			#address-cells = <1>;
+			#size-cells = <1>;
+			ranges = <0x0 0x2f0d0000 0x9000>;
+
+			/* Control region, with ICmsg buffers.
+			 * Size is fixed.
+			 */
+			cpuapp_cpucell_ipc_shm_ctrl: memory@0 {
+				reg = <0x0 0x1000>;
+			};
+			/* TX heap, user defined */
+			cpuapp_cpucell_ipc_shm_heap: memory@1000 {
+				reg = <0x1000 0x4000>;
+			};
+			/* RX heap, user defined */
+			cpucell_cpuapp_ipc_shm_heap: memory@5000 {
+				reg = <0x5000 0x4000>;
+			};
+		};
+
+		/* Shared memory ownership */
+		cpuapp_cpucell_ipc_shm: memory@2 {
+			compatible = "nordic,owned-memory";
+			reg = <0x2f0d0000 DT_SIZE_K(36)>;
+			owner-id = <2>;
+			perm-read;
+			perm-write;
+			status = "disabled";
+		};
+		cpucell_cpuapp_ipc_shm: memory@4 {
+			compatible = "nordic,owned-memory";
+			reg = <0x2f0d0000 DT_SIZE_K(36)>;
+			owner-id = <4>;
+			perm-read;
+			perm-write;
+			status = "disabled";
+		};
+
+		shared_ram20_region: memory@2f88f000 {
+			reg = <0x2f88f000 DT_SIZE_K(4)>;
+			#address-cells = <1>;
+			#size-cells = <1>;
+			ranges = <0x0 0x2f88f000 0x1000>;
+
+			cpuapp_cpusys_ipc_shm: memory@ce0 {
+				reg = <0xce0 0x80>;
+			};
+
+			cpusys_cpuapp_ipc_shm: memory@d60 {
+				reg = <0xd60 0x80>;
+			};
+
+			cpurad_cpusys_ipc_shm: memory@e00 {
+				reg = <0xe00 0x80>;
+			};
+
+			cpusys_cpurad_ipc_shm: memory@e80 {
+				reg = <0xe80 0x80>;
+			};
+		};
+
+		ram21_region: memory@2f890000 {
+			compatible = "nordic,owned-memory";
+			status = "disabled";
+			reg = <0x2f890000 DT_SIZE_K(32)>;
+			perm-read;
+			perm-write;
+			perm-secure;
+			#address-cells = <1>;
+			#size-cells = <1>;
+			ranges = <0x0 0x2f890000 0x8000>;
+
+			dma_fast_region: memory@4000 {
+				compatible = "zephyr,memory-region";
+				reg = <0x4000 DT_SIZE_K(16)>;
+				status = "disabled";
+				#memory-region-cells = <0>;
+				zephyr,memory-region = "DMA_RAM21";
+				zephyr,memory-attr = <( DT_MEM_DMA | DT_MEM_CACHEABLE )>;
+			};
+		};
+
+		cpuppr_ram3x_region: memory@2fc00000 {
+			compatible = "nordic,owned-memory";
+			reg = <0x2fc00000 DT_SIZE_K(16)>;
```

Comment:
For PPR code, 16 KB is not a lot. Should we increase it to at least 24?

Reply:
ok, increased